docs/user/environment-variables.md

```diff
@@ -4,19 +4,38 @@ The installer accepts a number of environment variable that allow the interactiv
 
 ## General
 
-| Environment Variable              | Description                                                                                 |
-|:----------------------------------|:--------------------------------------------------------------------------------------------|
-| `OPENSHIFT_INSTALL_BASE_DOMAIN`   | The base domain of the cluster. All DNS records will be sub-domains of this base.           |
-| `OPENSHIFT_INSTALL_CLUSTER_NAME`  | The name of the cluster. This will be used when generating sub-domains.                     |
-| `OPENSHIFT_INSTALL_EMAIL_ADDRESS` | The email address of the cluster administrator. This will be used to log in to the console. |
-| `OPENSHIFT_INSTALL_PASSWORD`      | The password of the cluster administrator. This will be used to log in to the console.      |
-| `OPENSHIFT_INSTALL_PLATFORM`      | The platform onto which the cluster will be installed.                                      |
-| `OPENSHIFT_INSTALL_PULL_SECRET`   | The container registry pull secret for this cluster.                                        |
-| `OPENSHIFT_INSTALL_SSH_PUB_KEY`   | The SSH key used to access all nodes within the cluster. This is optional.                  |
+* `OPENSHIFT_INSTALL_BASE_DOMAIN`:
+    The base domain of the cluster. All DNS records will be sub-domains of this base.
+
+    For AWS, this must be a previously-existing public Route 53 zone.  You can check for any already in your account with:
+
+    ```sh
+    aws route53 list-hosted-zones --query 'HostedZones[? !(Config.PrivateZone)].Name' --output text",
```

Comment:
there's an extra `",` at the end of this line that makes the command not copy/paste-able.

Reply:
> there's an extra `",` at the end

Oops, copy/paste from the Go version.  Fixed in 9324f35 -> fa7c9e3.